core/SConscript.boardloader

```diff
@@ -131,6 +131,7 @@ env.Replace(
     ] + CPPPATH_MOD,
     CPPDEFINES=[
         ('TREZOR_MODEL', '$TREZOR_MODEL'),
```

Comment:
We should remove these  
```suggestion
```


and only keep using `TREZOR_MODEL_X` defines

Reply:
We need to know the model value for the Qstr. But I suppose we can make it conditional on each model def, if there are no other usecases like that.

Reply:
> But I suppose we can make it conditional on each model def, if there are no other usecases like that.

Good point. Let's do it like this.